platform/location.hpp

```diff
@@ -70,6 +70,7 @@ namespace location
     double m_latitude;  //!< degrees
     double m_longitude; //!< degrees
     double m_speed;     //!< meters per second
+    double m_altitude;  //!< metres
```

Comment:
I don't see any reason to duplicate GpsInfo with GpsTrackInfo now when we decided to save detailed track data. GPX and KML can also store accuracy and bearing. Instead of copying the same functionality, it looks like GpsTrackInfo can be deleted completely and replaced by GpsInfo struct everywhere in a separate PR.

Reply:
See https://github.com/organicmaps/organicmaps/pull/9109